Uplift/Views/Footers/GymCellFooter.swift

```diff
@@ -23,56 +23,37 @@ class GymCellFooter: UIView {
         setupConstraints()
     }
 
-    func configure(for gym: Gym) {
-        locationNameLabel.text = gym.name
+    func configure(for fitnessCenter: FitnessCenter) {
 
-        //set gym status
-        let changingSoon = gym.isStatusChangingSoon()
+        locationNameLabel.text = fitnessCenter.name
 
-        if(gym.isOpen) {
-            statusLabel.textColor = changingSoon ? .accentOrange : .accentOpen
+        if fitnessCenter.isStatusChangingSoon() {
+            statusLabel.textColor = .accentOrange
+            print(fitnessCenter.isStatusChangingSoon())
         } else {
-            statusLabel.textColor = changingSoon ? .accentOrange : .accentClosed
+            statusLabel.textColor = fitnessCenter.isOpen() ? .accentOpen : .accentClosed
         }
-
-        statusLabel.text = gym.isOpen ? ClientStrings.CommonStrings.open : ClientStrings.CommonStrings.closed
-
-        // Set gym hours
-        hoursLabel.text = getHoursString(from: gym)
-
-        capacityStatusLabel.text = "Cramped"
-        capacityStatusLabel.textColor = .accentOrange
-        capacityCountLabel.text = "120/140"
-    }
-
-    private func getHoursString(from gym: Gym) -> String {
-        let now = Date()
-        let isOpen = gym.isOpen
-
-        let gymHoursToday = gym.gymHoursToday
-        let gymHoursTomorrow = gym.gymHours[now.getIntegerDayOfWeekTomorrow()]
-
-        let format: String
-
-        //if the gym has closed, else if it's still open
-        if now > gymHoursToday.closeTime {
-            //get the format for opening tomorrow
-            format = gymHoursTomorrow.openTime.getHourFormat()
-            //if the gym is also closed tomorrow, else if the gym is open tomorrow
-            if gym.closedTomorrow {
-                return ClientStrings.CommonStrings.closed
-            } else {
-                return ClientStrings.Home.gymDetailCellOpensAt + gymHoursTomorrow.openTime.getStringOfDatetime(format: format)
+        statusLabel.text = fitnessCenter.isOpen() ? ClientStrings.CommonStrings.open : ClientStrings.CommonStrings.closed
+
+        hoursLabel.text = fitnessCenter.getHoursString()
+
+        if let capacityStatus = fitnessCenter.getCapacityStatus(), let percentStr = fitnessCenter.getCapacityPercent() {
+            capacityStatusLabel.isHidden = false
+            capacityCountLabel.isHidden = false
+            capacityStatusLabel.text = capacityStatus.rawValue
+            capacityCountLabel.text = "\(percentStr) full"
+            switch capacityStatus {
+            case .Light:
```

Comment:
Is there a reason why these cases are capitalized or am I tripping

Reply:
I made them caps so I can use the `capacityStatus.rawValue` without having to define a second set of upper case strings. I can change that back if you would like?

Reply:
Ohhh okie no worries, that's fine